app/assets/javascripts/controllers/middleware_jdbc_driver/middleware_jdbc_driver_controller.js

```diff
@@ -29,10 +29,10 @@ function MwAddJdbcDriverController($scope, $http, miqService) {
     })
       .then(
         function(result) { // success
-          miqService.miqFlash(result.data.status, result.data.msg);
+          miqService.replacePartials(result.data);
         },
         function() { // error
-          miqService.miqFlash('error', 'Unable to add JDBC Driver "' + data.driverName + '" on this server.');
+          miqService.replacePartials(result.data);
```

Comment:
This  uses the JSON returned by the helper `javascript_flash`  and replace the flash partial with the new html returned.

This is a JSON example.

```
{
	"explorer": "flash",
	"replacePartials": {
		"flash_msg_div": " <html here> "
}
```